app/globals.css

```
@@ -255,6 +255,37 @@
+
+.react-flow__node.rf-node-warning [data-slot="card"] {
+  box-shadow: 0 0 0 2px color-mix(in oklch, oklch(0.78 0.15 75) 85%, transparent), 0 0 12px -2px oklch(0.78 0.15 75);
+}
+
+/* One-shot pulse when the inspector scrolls to a field after clicking an
+   audit error (see FieldIssueWrap in inspector-panel). */
+.field-flash {
+  animation: fieldFlash 1.1s ease-out 1;
+}
+
+@keyframes fieldFlash {
```

Stylelint (17.12.0)

[error] Expected keyframe name "fieldFlash" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)